frontstage/templates/register/register.enter-your-details.html

```diff
@@ -126,6 +126,7 @@ <h2 class="section__title neptune u-mt-m">Phone number</h2>
                     {{ form.phone_number.label(class_='label') }}
                     {{ form.phone_number(class_='input input--text input-type__input') }}
                 </div>
+                <span style="font-size: 80%">For international numbers include the country code</span>
```

Comment:
Is there a CSS class we could use for this? I don't like us putting in in-line styles.

Reply:
Agreed, thanks. I've added a class ".field-description", which is bound to be useful for other things.